election/models.py

```diff
@@ -14,8 +14,8 @@ class Election(RandomPrimaryIdModel):
     candidates = ArrayField(models.CharField("Name", max_length=255))
     on_invitation_only = models.BooleanField(default=False)
     num_grades = models.PositiveSmallIntegerField("Num. grades", null=False)
-    start_at = models.IntegerField("Start date", default=round(time()))
-    finish_at = models.IntegerField("End date",default=round(time()+1))
+    start_at = models.IntegerField("Start date", default=1587059214 )
+    finish_at = models.IntegerField("End date",default=1587059215 )
```

Comment:
I see what you are doing but I don't think it is the proper way. I think we should just put a pointer to a function, instead of a value. See https://stackoverflow.com/questions/2029295/django-datefield-default-options (they put `date.today` for example instead of `date.today()`)